src/main/java/uk/gov/hmcts/darts/usermanagement/service/impl/UserManagementServiceImpl.java

```diff
@@ -135,6 +135,11 @@ private UserAccountEntity updatedUserAccount(UserPatch userPatch, UserAccountEnt
     }
 
     private void updateEntity(UserPatch userPatch, UserAccountEntity userAccountEntity) {
```

Comment:
are we confiming the formst of the email address anywhere to ensure its a valid email address?